templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>

```

Comment:
The anchor tags currently have placeholder `#` URLs. Replace these with actual URLs to the respective pages in your application to make the navigation functional.